cypress/pom/accountPage.ts

```diff
@@ -3,30 +3,30 @@ import { BasePage } from "./base/basePage";
 
 export class AccountPage extends BasePage {
   constructor() {
-    super('#my-account', 'AccountPage', 'index.php?controller=my-account')
+    super('#my-account', 'AccountPage', 'index.php?controller=my-account');
   }
 
   private get userInfo(): Cypress.Chainable {
-    return this.container.find('a[title = "View my customer account"]', { timeout: 10000 })
+    return this.container.find('a[title = "View my customer account"]', { timeout: 10000 });
   }
 
   private get bodyText(): Cypress.Chainable {
-    return this.container.find('#center_column', { timeout: 10000 })
+    return this.container.find('#center_column', { timeout: 10000 });
   }
 
-  public checkUserInfo(): this {
-    cy.allure().startStep(`Check user Info in header`)
+  public checkUserInfo(loginName): this {
```

Comment:
Don’t  forget to use types! In this case you must define function parameters like: 
`string: loginName`